Validation/Geometry/interface/MaterialBudgetHcal.h

```diff
@@ -37,6 +37,7 @@ class MaterialBudgetHcal : public SimWatcher,
   std::unique_ptr<MaterialBudgetHcalHistos> theHistoHcal_;
   std::unique_ptr<MaterialBudgetCastorHistos> theHistoCastor_;
   double rMax_, zMax_;
+  bool useDD4Hep_;
```

Comment:
Isn't our standard that this variable (and those like it) should be `fromDD4hep_`?

Reply:
changed